frontend/src/components/common/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import styles from './styles.module.scss';
+
+interface CheckboxType {
+  options: Array<string>;
+  onChange(value: string): void;
+}
+
+export const Checkbox: React.FC<CheckboxType> = ({ onChange, options }) => {
+  return (
```

Comment:
Avoid return if you don't need any logic in the component body

Reply:
Did you mean I should write like that?
```
export const Checkbox: React.FC<CheckboxType> = ({ onChange, options }) => (
  <div>
    ...
  </div>
);
```

Reply:
Yes, you are right 👍